Mathlib/Data/Complex/Basic.lean

```diff
@@ -639,41 +712,45 @@ theorem inv_def (z : ℂ) : z⁻¹ = conj z * ((normSq z)⁻¹ : ℝ) :=
 #align complex.inv_def Complex.inv_def
 
 @[simp]
-theorem inv_re (z : ℂ) : z⁻¹.re = z.re / normSq z := by simp [inv_def, division_def]
+theorem inv_re (z : ℂ) : z⁻¹.re = z.re / normSq z := by simp [inv_def, division_def, ofReal']
 #align complex.inv_re Complex.inv_re
 
 @[simp]
-theorem inv_im (z : ℂ) : z⁻¹.im = -z.im / normSq z := by simp [inv_def, division_def]
+theorem inv_im (z : ℂ) : z⁻¹.im = -z.im / normSq z := by simp [inv_def, division_def, ofReal']
 #align complex.inv_im Complex.inv_im
 
-@[simp, norm_cast]
-theorem of_real_inv (r : ℝ) : ((r⁻¹ : ℝ) : ℂ) = r⁻¹ :=
-  ext_iff.2 <| by simp
-#align complex.of_real_inv Complex.of_real_inv
+-- removed norm-cast
```

Comment:
Comments like this really need `porting note` in them. Why did you remove it?

Reply:
Accidentally did not undo the remove `norm_cast` after coercions added.